backend/kesaseteli/applications/exporters/excel_exporter.py

```diff
@@ -31,16 +31,15 @@ class ExcelField(NamedTuple):
 SPECIAL_CASE_FIELD_TITLE = _("Erikoistapaus (esim yhdeksäsluokkalainen)")
 SUM_FIELD_TITLE = _("Summa")
 WORK_HOURS_FIELD_TITLE = _("Työtunnit")
+INVOICER_EMAIL_FIELD_TITLE = _("Laskuttajan sähköposti")
+INVOICER_NAME_FIELD_TITLE = _("Laskuttajan nimi")
+INVOICER_PHONE_NUMBER_FIELD_TITLE = _("Laskuttajan Puhelin")
 
 REMOVABLE_REPORTING_FIELD_TITLES = [
     _("Y-tunnus"),
     _("Yhdyshenkilö"),
-    _("Yhdyshenkilön sähköposti"),
-    _("Yhdyshenkilön Puhelin"),
     _("Erillinen laskuttaja"),
-    _("Laskuttajan nimi"),
-    _("Laskuttajan sähköposti"),
-    _("Laskuttajan Puhelin"),
+    INVOICER_NAME_FIELD_TITLE,
     _("Raporttiin luokittelu"),
```

Comment:
Is it on purpose you are just adding back the name and not the email and phone number?